tests/unit_tests/test_game.py

```diff
@@ -52,6 +52,56 @@ def test_instance_logging(game_stats_service):
     await game.validate_game_settings()
     assert game.validity is ValidityState.VALID
 
+async def test_ffa_not_rated(game_service, game_stats_service):
+    game = Game(6, game_service, game_stats_service)
+    game.state = GameState.LOBBY
+    players = [
```

Comment:
I'm aware I'm responsible for this duplication, but we should keep in mind that test code should be maintainable as well. A refactor into a reusable fixture of sorts would be much apreciated :)